src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorStringsImporter.kt

```diff
@@ -80,18 +82,41 @@ object PoEditorStringsImporter {
                               defaultLang: String,
                               resDirPath: String,
                               tags: List<String>?,
-                              languageValuesOverridePathMap: Map<String, String>?) {
+                              languageValuesOverridePathMap: Map<String, String>?,
+                              minimumTranslationPercentage: Int?) {
```

Comment:
If you apply the suggestion that I mentioned in `PoEditorPlugin` you can make this non-nullable.